react/forms/step2.md

```diff
@@ -0,0 +1,122 @@
+In PatternFly 4, demos are used to show how Patternfly’s components and layouts can be put together to build more complex structures.
+
+They're made entirely from components and layouts.
```

Comment:
This line / topic is somewhat odd.

Perhaps some rewording to speak more about why / how forms are used, less about PatternFly demos?